docs/getting-started/installing/troubleshooting.md

```diff
@@ -4,21 +4,18 @@ title: Troubleshooting
 
 # Services don't start
 
-## Symptons
+## Symptoms
 
 1. `kontena node ls` show nodes in a __OK__ status, but services are not starting.
-2. `kontena grid logs` show execeptions like the one below
+2. `kontena grid logs` show execeptions that looks like the one below
 
```

Comment:
I think this commands will return that grid is online and node is online. Did you mean change the first command from `kontena node ls` to `kontena grid health`?